hack/tools/release/notes/main.go

```diff
@@ -187,12 +187,6 @@ func validateConfig(config *notesCmdConfig, releaseType string) error {
 		}
 	}
 
-	if releaseType != "" {
```

Comment:
why is this going away?

Reply:
As `previousReleaseVersion` is not required always, it only need to be set when previous version is available, so we don't need this check. 
Also with this check we won't be able to create initial RC release, will get error like below
```bash
RELEASE_TAG=v1.8.0-rc.0 make release-notes
```